CHANGELOG.md

```diff
@@ -11,37 +11,21 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - `Tileset::columns`.
 - `Layer::id`, `Layer::width` and `Layer::height`.
 - Support for 'object'-type properties.
-- Documentation for map members.
-- Tests for `tiled_base64_zstandard.tmx`.
-- `.gitattributes` for line ending consistency.
```

Comment:
Why'd you remove these additions?

Reply:
I think the addition of documentation or especially `.gitattributes` or fixing of markdown formatting is not noteworthy for people who want to know what changed in rs-tiled, so I would not put them in the changelog. If people want to know all the details, they could just look at the git history.

But I'll put them back for now, since their removal was unrelated to this change. Let me know your opinion though, since I'd still like to remove these.